scripts/SMD_chip_package_rlc-etc/size_definitions/size_capacitor_tantal.yaml

```diff
@@ -1,4 +1,4 @@
-SMD_1608-08:
+C_1608-08:
```

Comment:
There is no `1608-08` in that datasheet. Only `1608-09`. The difference is the height.

Reply:
I'm thinking the datasheet may have changed over time. Look at the dimensions for the 1608-10 in the file comment and compare with the latest datasheet. Two mistakes are too many to consider as simply a typo IMO. I reckon it makes sense to update this to the current 1608-09 in the datasheet and leave the 1608-08 footprint in our library for now, removing it in the v6 library. OK?

Reply:
A quick search shows that both `1608-08` and `1608-09` are sold.
I am resolving this conversation to reduce noise now. Its part of the `tantal` file that we should discuss as a whole in dedicated PR.